VTEX - Logistics API.json

```diff
@@ -7487,148 +7353,43 @@
                                             "properties": {
                                                 "dayOfWeek": {
                                                     "type": "integer",
-                                                    "format": "int32"
+                                                    "format": "int32",
+                                                    "description": "Day of the week identification, as in `1` = Monday, `2` = Tuesday, `3` = Wednesday, `4` = Thursday, and `5` = Friday.",
```

Comment:
Do Saturday and Sunday work as well?

Reply:
I suggest adding an enum to describe all possible values

Reply:
Saturday and Sunday do not work, it is a configuration for days of the week.